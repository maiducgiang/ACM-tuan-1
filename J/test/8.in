33
667,361(323)
390,642(617)
644,860(560)
655,809(721)
250,849(957)
972,282(797)
637,562(669)
834,845(604)
196,718(973)
471,326(480)
385,751(649)
720,922(812)
269,241(317)
810,108(377)
909,708(444)
460,352(890)
400,506(226)
861,157(935)
523,561(315)
165,341(112)
165,927(601)
814,107(588)
490,967(881)
417,794(179)
238,258(573)
387,537(457)
363,937(932)
424,229(238)
466,572(672)
515,849(569)
727,214(691)
210,1000(223)
145,550(516)
