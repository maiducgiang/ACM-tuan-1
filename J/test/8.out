333346981 499500
15610079 39960
6442157 9990
81894239 124875
20883259 83250
194262103 199800
212308369 333000
834010759 999000
39304451 199800
78475859 166500
192682949 499500
72020189 99900
67243019 249750
809298269 999000
1023422 1125
229946269 499500
10002643 24975
430148389 499500
29057653 55500
165175771 999000
82880837 499500
271097827 333000
245238457 499500
83475277 199800
15868021 66600
9678748 24975
72714799 199800
423805009 999000
1553687 3330
12883343 24975
726487477 999000
2098899223 9990000
24234161 166500
